### PR TITLE
Add new built-in plugin to work with AWS EBS

### PR DESCRIPTION
**Pull request by a1exsh**
Currently only possible to list EBS snapshots.

* Primary use-case: alert on stale snapshots.
* Code skeleton copied over from S3 wrapper.
* Extracted get_region() function into a common file.

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/ebs.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        """
+        return propartial(EBSWrapper, region=factory_ctx.get('entity').get('region', None))
+
+
+class EBSWrapper(object):
+
+    def __init__(self, region=None):
+        if not region:
+            region = get_instance_identity_document()['region']
+        self.__client = boto3.client('ebs', region_name=region)
```
Comment (a1exsh) [inline zmon_worker_monitor/builtins/plugins/ebs.py]: Typo: `s/ebs/ec2/` :-(

**Comment by a1exsh** [comment]
:+1:

**Comment by a1exsh** [comment]
Now everything seems to work.

**Comment by lmineiro** [comment]
👍

**Comment by lmineiro** [comment]
Want to send an update to the docs?

**Comment by a1exsh** [comment]
Ah, is there a separate doc repo?

**Comment by lmineiro** [comment]
Yes: https://github.com/zalando-zmon/zmon-docs

**Pull request merged**